web/app/controller/Root.js

```diff
@@ -52,7 +52,7 @@ Ext.define('Traccar.controller.Root', {
         if (value !== undefined) {
             return Traccar.AttributeFormatter.getAttributeConverter(this.attributeKey)(value);
         }
-        return null;
+        return undefined;
```

Comment:
Undefined value shouldn't be used explicitly. I propose we remove return statement instead.